### DISCUSSION TITLE
Hi, how can i choose thorium version compiled by AVX2 or SSE3

### DISCUSSION THREAD

**Discussion by thiendt2k1** [Alex313031/Thorium-Win#155] [Q&A] [answered]
Hi, i am trying to decide whether to use AVX2 or SSE3 for my Intel I3 - 12100f, it said here they include sse4 and avx2, is sse3 still perform ok on 4, or should i just go with AVX2 for safety?
![image](https://github.com/Alex313031/Thorium-Win/assets/58879301/20a1a792-5163-43e0-b36d-7e11acea5595)
TIA.

**Comment by Unknown78** [comment, marked as answer]
Someone validate it with CPU-Z
https://valid.x86.fr/amfi0f

Just try AVX2 first.
